misc/ninja_syntax.py

```diff
@@ -21,8 +21,11 @@ def __init__(self, output, width=78):
     def newline(self):
         self.output.write('\n')
 
-    def comment(self, text):
-        for line in textwrap.wrap(text, self.width - 2):
+    def comment(self, text, has_path=False):
+        args = {}
+        if has_path:
```

Comment:
Should this just be done unconditionally? i.e. always pass break_long_words=False, break_on_hyphens=False ? I'm not sure, but I can't think of any case where you wouldn't want this, and is most callers will end up passing True for has_path over time, and the ones that don't will likely be incorrect.